lib/opensusebasetest.pm

```diff
@@ -103,8 +107,8 @@ done", "binaries-with-missing-libraries.txt", {timeout => 60, noupload => 1});
         clear_console;
     }
 
-    script_run 'tar cvvJf problem_detection_logs.tar.xz *';
-    upload_logs('problem_detection_logs.tar.xz');
+    script_run 'tar cvvJf problem_detection.tar.xz *';
+    upload_logs('problem_detection.tar.xz');
```

Comment:
I love the possibility to be able to [get to know those cores](https://www.youtube.com/watch?v=YlFXCC4Hfa0)... But back to being serious, 'logs_and_coredumps.xz' sounds great :)